modules/Bio/EnsEMBL/AssemblyMapper.pm

```diff
@@ -223,7 +223,7 @@ sub map {
   throw('Incorrect number of arguments.') if (!( @_ >= 6));
 
   my ( $self, $frm_seq_region_name, $frm_start, $frm_end, $frm_strand,
-       $frm_cs, $to_slice )
+       $frm_cs, $dummy, $to_slice, $include_org_coord )
```

Comment:
The documentation doesn't match the new function prototype, for example what this $dummy parameter is.